mindee/fields/position.py

```diff
@@ -10,7 +10,7 @@
 )
 
 
-class Position(BaseField):
+class PositionField(BaseField):
```

Comment:
Comments ?

Reply:
Added for properties, class itself doesn't  need to be documented as it's not considered public (init by end-user).